wordpress-mysql-replication/nested/mysql-replication.json

```diff
@@ -1,551 +1,548 @@
-      }
-    },
-    "dbSubnetStartAddress": {
-      "type": "string",
-      "defaultValue": "10.0.1.4",
-      "metadata": {
-        "description": "Start IP address in the subnet for the VMs"
-      }
-    },
-    "imagePublisher": {
-      "type": "string",
-      "defaultValue": "OpenLogic",
-      "allowedValues": [
-        "OpenLogic"
-      ],
-      "metadata": {
-        "description": "publisher for the VM OS image"
-      }
-    },
-    "imageOffer": {
-      "type": "string",
-      "defaultValue": "CentOS",
-      "allowedValues": [
-        "CentOS"
-      ],
-      "metadata": {
-        "description": "VM OS name"
-      }
-    },
-    "imageSKU": {
-      "type": "string",
-      "defaultValue": "6.6",
-      "allowedValues": [
-        "6.5",
-        "6.6"
-      ],
-      "metadata": {
-        "description": "VM OS version"
-      }
-    },
-    "mysqlFrontEndPort0": {
-      "type": "int",
-      "defaultValue": 3306,
-      "metadata": {
-        "description": "MySQL public port master"
-      }
-    },
-    "mysqlFrontEndPort1": {
-      "type": "int",
-      "defaultValue": 3307,
-      "metadata": {
-        "description": "MySQL public port slave"
-      }
-    },
-    "sshNatRuleFrontEndPort0": {
-      "type": "int",
-      "defaultValue": 64001,
-      "metadata": {
-        "description": "public ssh port for VM1"
-      }
-    },
-    "sshNatRuleFrontEndPort1": {
-      "type": "int",
-      "defaultValue": 64002,
-      "metadata": {
-        "description": "public ssh port for VM2"
-      }
-    },
-    "mysqlProbePort0": {
-      "type": "int",
-      "defaultValue": 9200,
-      "metadata": {
-        "description": "MySQL public port master"
-      }
-    },
-    "mysqlProbePort1": {
-      "type": "int",
-      "defaultValue": 9201,
-      "metadata": {
-        "description": "MySQL public port slave"
-      }
-    },
-    "artifactsPath": {
-      "type": "string",
-      "metadata": {
-        "description": "template and script file location",
-        "artifactsBaseUrl": "Base URL of the Publisher Template gallery package"
-      }
-    },
-    "customScriptCommandToExecute": {
-      "type": "string",
-      "defaultValue": "bash azuremysql.sh",
-      "metadata": {
-        "description": "bash script command line"
-      }
-    },
-    "publicIPName": {
-      "type": "string",
-      "defaultValue": "mysqlIP01",
-      "metadata": {
-        "description": "public IP name for MySQL loadbalancer"
-      }
-    },
-    "storageAccountName": {
-      "type": "string",
-      "metadata": {
-        "description":  "storage account name"
+﻿{  
+   "$schema":"http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+   "contentVersion":"1.0.0.0",
+   "parameters":{  
+      "dnsName":{  
+         "type":"string",
+         "metadata":{  
+            "description":"Connect to your cluster using dnsName.location.cloudapp.azure.com"
```

Comment:
this won't be accurate outside of public azure - since FQDN is an output probably not needed